src/System.Memory/tests/MemoryMarshal/TryGetArray.cs

```diff
@@ -46,5 +46,44 @@ public static void OwnedMemoryTryGetArray()
                 Assert.Equal(array[i], segment.Array[i]);
             }
         }
+
+        [Fact]
+        public static void TryGetArrayFromEmptyReadOnlyMemory()
+        {
+            Assert.True(MemoryMarshal.TryGetArray(ReadOnlyMemory<byte>.Empty, out ArraySegment<byte> segment));
+            Assert.Equal(0, segment.Array.Length);
+        }
+
+        [Fact]
+        public static void TryGetArrayFromEmptyMemory()
+        {
+            int[] array = new int[0];
+            Memory<int> memory = array;
```

Comment:
nit: Though `Memory<T>` gets implicitly casted to ReadOnlyMemory, I think we should explicitly use ReadOnlyMemory here.

Reply:
Done